app/src/main/java/com/digitalarchitects/rmc_app/data/editmyaccount/EditMyAccountUIEvent.kt

```diff
@@ -1,13 +1,19 @@
 package com.digitalarchitects.rmc_app.data.editmyaccount
 
+import com.digitalarchitects.rmc_app.data.myrentals.MyRentalsUIEvent
 import com.digitalarchitects.rmc_app.model.UserType
 
 sealed interface EditMyAccountUIEvent {
```

Comment:
Waarom heb je gekozen voor een interface ipv een sealed class?

Reply:
Ik verwacht dat Brian af wil dwingen dat je alle events implementeert?

Reply:
Ik zie nu dat als je sealed class gebruikt je ook exhaustive moet zijn met je onEvent functie in de ViewModel.

Reply:
Een best practice is om een sealed interface te gebruiken als er geen constructor nodig is. Scheelt code schrijven en kost minder geheugen. Verder is het een kwestie van waar wil je dat de implementatie plaatsvindt, voor de navigatie past dit prima in de onEvent in viewmodel zelf.
ZIe ook: https://www.youtube.com/watch?v=kLJRZpRhX1o&ab_channel=PhilippLackner

Reply:
Nice, dat wist ik niet. Lijkt me goed om dit dan voor alle UIEvents gelijk te trekken. 👍